interface/resources/qml/AssetServer.qml

```diff
@@ -0,0 +1,208 @@
+//
+//  AssetsServer.qml
+//
+//  Created by Clement on  3/1/16
+//  Copyright 2016 High Fidelity, Inc.
+//
+//  Distributed under the Apache License, Version 2.0.
+//  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
+//
+
+import QtQuick 2.5
+import QtQuick.Controls 1.4
+import QtQuick.Dialogs 1.2 as OriginalDialogs
+import Qt.labs.settings 1.0
+
+import "styles-uit"
+import "controls-uit" as HifiControls
+import "windows-uit"
+import "dialogs"
+
+Window {
+    id: root
+    objectName: "AssetsServer"
+    title: "Assets Server"
```

Comment:
I'm not sure this title makes sense - shouldn't this be "Asset Server" or just "Assets"?